tests/e2e/support/api/token/utils.ts

```diff
@@ -0,0 +1,88 @@
+import { TokenEnvironmentFactory } from '../../environment'
+import { config } from '../../../config'
+import fetch from 'node-fetch'
+import { User } from '../../types'
+const logonUrl = config.backendUrl + '/signin/v1/identifier/_/logon'
+const redirectUrl = config.backendUrl + '/oidc-callback.html'
+const tokenUrl = config.backendUrl + '/konnect/v1/token'
+const getAuthorizedEndPoint = async (user) => {
+  const logonResponse = await fetch(logonUrl, {
+    method: 'POST',
+    headers: {
+      'Kopano-Konnect-XSRF': '1',
+      Referer: config.backendUrl,
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      params: [user.id, user.password, '1'],
+      hello: {
+        scope: 'openid profile email',
+        client_id: 'web',
+        redirect_uri: redirectUrl,
+        flow: 'oidc'
+      }
+    })
+  })
+  if (logonResponse.status !== 200) {
+    throw new Error(`logonResponse.status is ${logonResponse.status}, expected 200`)
+  }
+  const cookies = logonResponse.headers.raw()['set-cookie']
+  const data = await logonResponse.json()
+  const authorizedUrl = data.hello.continue_uri
+  return [authorizedUrl, cookies]
+}
+
+const getAuthorizedToken = async (user, continueUrl, cookies) => {
```

Comment:
```suggestion
const getCode = async (user, continueUrl, cookies) => {
```

Reply:
may be how about `getAuthorizationCode`?